easybuild/easyblocks/d/doxygen.py

```diff
@@ -30,30 +30,32 @@
 @author: Kenneth Hoste (Ghent University)
 @author: Pieter De Baets (Ghent University)
 @author: Jens Timmerman (Ghent University)
+@author: Balazs Hajgato (Free University Brussels (VUB))
 """
 
-from easybuild.easyblocks.generic.configuremake import ConfigureMake
+from distutils.version import LooseVersion
 from easybuild.tools.run import run_cmd
+from easybuild.easyblocks.generic.cmakemake import CMakeMake
 
-
-class EB_Doxygen(ConfigureMake):
+class EB_Doxygen(CMakeMake):
     """Support for building/installing Doxygen"""
 
     def configure_step(self):
-        """Configure build using non-standard configure script (see prefix option)"""
+        if LooseVersion(self.version) < LooseVersion("1.8.10"):
+             """Configure build using non-standard configure script (see prefix option)"""
```

Comment:
@boegel was without `=` 
the normal is `--prefix=foo/bar` and here is `--prefix foo/bar`